app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupingSettingsViewHolder.java

```diff
@@ -0,0 +1,68 @@
+package com.randomappsinc.studentpicker.grouping;
+
+import android.view.View;
+import android.widget.EditText;
+
+import com.randomappsinc.studentpicker.R;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+class GroupingSettingsViewHolder {
+
+    @BindView(R.id.num_of_names_in_group) EditText namesInGroup;
+    @BindView(R.id.num_of_groups) EditText numGroups;
+
+    private GroupingSettings settings;
+
+    GroupingSettingsViewHolder(View view, GroupingSettings settings) {
+        ButterKnife.bind(this, view);
+        this.settings = settings;
+    }
+
+    void revertSettings() {
+        namesInGroup.setText(String.valueOf(settings.getNumOfNamesInGroup()));
+        namesInGroup.clearFocus();
+        numGroups.setText(String.valueOf(settings.getNumOfGroups()));
+        numGroups.clearFocus();
+    }
+
+    void applySettings() {
+        applyNumberOfNames(namesInGroup);
+        applyNumberOfGroups(numGroups);
+    }
+
+    private void applyNumberOfNames(EditText namesInGroup) {
+        String numOfNamesInGroup = namesInGroup.getText().toString();
```

Comment:
nit: I know that the inputType is digits only, but I feel like being extra safe here doesn't hurt.
```suggestion
        String numOfNamesInGroup = namesInGroup.getText().toString().trim();
```